src/Element/Attributes.elm

```diff
@@ -533,3 +537,8 @@ aspectRatio value =
 direction : String -> Attribute msg
 direction value =
     stringYogaProperty "direction" value
+
+
+checked : Bool -> Attribute msg
+checked value =
```

Comment:
After thinking more, this should probably be `isOn`. Seems like the intention is to follow the iOS apis where possible?